### PR TITLE
feat:DE-9: add member schema

### PR DESCRIPTION
**Pull request by Aradhya-successive**

### CODE REVIEW

File: src/repositories/member/MemberRepository.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  IMemberDocument,
+  mongoose.Model<IMemberDocument>
+> {
+  constructor() {
+    super(memberModel);
+  }
+  public static generateObjectId() {
+    return String(mongoose.Types.ObjectId());
+  }
+  public create(data: any): Promise<IMemberDocument> {
```
Comment (Nikhil-Rawat) [inline src/repositories/member/MemberRepository.ts:17]: Move originalID to versioning

File: src/repositories/member/MemberModel.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import * as mongoose from "mongoose";
+import IMemberDocument from "./IMemberDocument";
+import MemberSchema from "./MemberSchema";
+
+export const memberSchema = new MemberSchema({ collections: "member" });
+
```
Comment (barkhatyagi) [inline src/repositories/member/MemberModel.ts:6]: add interface

File: src/repositories/member/IMemberDocument.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import IVersionalbleDocument from "../versionable/IDocument";
+
```
Comment (Nikhil-Rawat) [inline src/repositories/member/IMemberDocument.ts:2]: Add comment

**Review by Nikhil-Rawat — CHANGES_REQUESTED** [review]
Add comment

**Review by Nikhil-Rawat — APPROVED** [review]

**Review by barkhatyagi — APPROVED** [review]

**Pull request merged**